src/Symfony/Component/Lock/Bridge/DynamoDb/composer.json

```diff
@@ -0,0 +1,40 @@
+{
+    "name": "symfony/amazon-dynamodb-lock",
+    "type": "symfony-lock-bridge",
+    "description": "Symfony Amazon DynamoDb extension Lock Bridge",
+    "keywords": [],
+    "homepage": "https://symfony.com",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Fabien Potencier",
+            "email": "fabien@symfony.com"
+        },
+        {
+            "name": "Symfony Community",
+            "homepage": "https://symfony.com/contributors"
+        }
+    ],
+    "require": {
+        "php": ">=8.2",
+        "async-aws/core": "^1.7",
+        "async-aws/dynamo-db": "^3.0",
+        "symfony/lock": "^7.3",
+        "symfony/service-contracts": "^2.5|^3",
+        "psr/log": "^1|^2|^3"
+    },
+    "require-dev": {
+        "symfony/polyfill-uuid": "^1.15",
+        "symfony/http-client-contracts": "^2.5|^3"
+    },
+    "conflict": {
+        "symfony/http-client-contracts": "<2.5"
+    },
```

Comment:
Looking at the `DynamoDbStore` class, I don't get why `symfony/service-contracts`, `psr/log`, `symfony/polyfill-uuid` and `symfony/http-client-contracts` need to be explicitly listed as dependencies.

Reply:
@GromNaN that's a miss from when I copied the files structure from the Lock component. 
I've removed most of the packages you listed, however I did explicitly add `symfony/polyfill-uuid` to `^1.15` as otherwise it would default to `1.13` and this version is failing as it's looking for a constant that doesn't exist and was breaking the CI.

Maybe I'm missing something and there is a better way here?

Reply:
What error do you get exactly ?

Is this `async-aws/dynamodb` using a uuid feature that is not yet polyfilled in older version of the polyfill package, or is it the polyfill package itself being broken ?

Reply:
The constants were not correctly declared in 1.13.0: ([diff](https://github.com/symfony/polyfill-uuid/compare/v1.13.0...v1.14.0#diff-d11e2b9e464a2d6711842693ee58d997e7d238e803fbbe33cea332ece518159eL15-L25)). This blocks PHPUnit execution.

```
PHPUnit 9.6.22 by Sebastian Bergmann and contributors.

Error in bootstrap script: Error:
Undefined constant "UUID_VARIANT_NCS"
#0 src/Symfony/Component/Lock/Bridge/DynamoDb/vendor/composer/autoload_real.php(41): require()
```

Reply:
@GromNaN beat me to it, was sleeping on this side of the world 😄 

You can see this failure in this [run](https://github.com/symfony/symfony/actions/runs/14350059000/job/40226931778#step:9:10669)

@stof Thanks for that, and for the PR in async-aws